src/stats.jl

```diff
@@ -391,3 +391,25 @@ end
 mean(chn::Chains, syms) = mean(chn[:, syms, :])
 # resolve method ambiguity with `mean(f, ::AbstractArray)`
 mean(chn::Chains, syms::AbstractVector) = mean(chn[:, syms, :])
+
+
+
+"""
+    MCMCDiagnosticTools.bfmi(chains::Chains)
+
+Compute the Bayesian Fraction of Missing Information (BFMI) for a set of chains.
+
+This is a diagnostic for Hamiltonian Monte Carlo (HMC) methods, which indicates how well the
+    momentum resampling explores the energy distribution.
+
+Returns a Vector{Float64} of BFMI values, one for each chain.
+"""
+function MCMCDiagnosticTools.bfmi(chains::Chains)
+
+    if :hamiltonian_energy ∉ names(chains)
+        throw(ArgumentError("The chain does not contain the `:hamiltonian_energy` parameter. 
```

Comment:
These sorts of long strings can be a bit tricky to make fit into line length limits. There are a few ways to do it, my favourite is using tripe-quoted strings: https://docs.julialang.org/en/v1/manual/strings/#Triple-Quoted-String-Literals